docs/src/getting_started/optim_compatibility.md

```diff
@@ -0,0 +1,188 @@
+# Optimizing Scene Parameters using Optim.jl
+
+In this tutorial we will explore the exact same problem as demonstrated
+in [Inverse Lighting Tutorial](@ref) but this time we will use the
+Optimization Package [Optim.jl](https://julianlsolvers.github.io/Optim.jl/stable/).
+I would recommend going through a few of the
+[tutorials on Optim](https://julianlsolvers.github.io/Optim.jl/stable/#user/minimization/#_top)
+before starting this one.
+
+If you have already read the previous tutorial, you can safely skip to
+[Writing the Optimization Loop using Optim](@ref). The part previous to this
+is same as the previous tutorial.
+
+```julia
+using RayTracer, Images, Zygote, Flux, Statistics, Optim
+```
+
+## Configuring the Scene
```

Comment:
Is it useful to duplicate this part? I would just put the bare code, without real explanations, but a reference to the previous one for the details.